### DISCUSSION TITLE
Need A nodejs code

### DISCUSSION THREAD

**Discussion by amkfd1** [directus/directus#13570] [Archive]
Could you please upload a nodejs implementation code. I'd like access to my database-fields and contents.

**Comment by rijkvanzanten** [comment]
Are you asking for an example hook in particular, or...?

**Comment by amkfd1** [comment]
Actually we've figured it out. Thank you!! 
Thursday, 26 May 2022, 03:07pm +01:00 from Rijk van Zanten  ***@***.*** :
